lib/jnpr/junos/facts/routing_engines.py

```diff
@@ -8,7 +8,10 @@ def facts_routing_engines(junos, facts):
         'model',
         'up-time',
         'last-reboot-reason']
-    re_info = junos.rpc.get_route_engine_information()
+    try:
+        re_info = junos.rpc.get_route_engine_information()
+    except:
+        re_info = junos.rpc.get_route_engine_information(infrastructure="FM-0")
```

Comment:
Can we have 1-2 line description (doc string) above this line of code. This is very specific to Qfabric setup only. 
So can you explain this itself.